go/arrow/internal/flatbuf/Binary.go

```diff
@@ -22,7 +22,7 @@ import (
 	flatbuffers "github.com/google/flatbuffers/go"
 )
 
-/// Opaque binary data
+// / Opaque binary data
```

Comment:
Auto formatter noise? How can we prevent this?

Reply:
I'll revert most of these